src/NumericalIntegration.jl

```diff
@@ -71,6 +126,90 @@ function integrate(x::AbstractVector, y::AbstractVector, ::SimpsonEvenFast)
     @inbounds return (x[2] - x[1]) * retval
 end
 
+"""
+    integrate(x::AbstractVector, y::AbstractArray, method)
+
+When y is an array, compute integral for each column.
+"""
+function integrate(x::AbstractVector, y::AbstractArray, M::IntegrationMethod)
```

Comment:
Why columns in particular? I think this should take an `axis` argument that can default to columns to keep with the MATLAB convention.

Reply:
Yep, good point. I can do that. I chose the keyword term "dims" to make it consistent with some of the Base functions (like sum() etc).